tools/benchmark.sh

```diff
@@ -696,7 +1061,7 @@ for job in ${jobs[@]}; do
     echo "Completed $job (ID: $job_id) in $((end-start)) seconds" | tee -a $schedule
   fi
 
-  echo -e "ops/sec\tmb/sec\tSize-GB\tL0_GB\tSum_GB\tW-Amp\tW-MB/s\tusec/op\tp50\tp75\tp99\tp99.9\tp99.99\tUptime\tStall-time\tStall%\tTest\tDate\tVersion\tJob-ID"
+  echo -e "ops_sec\tmb_sec\tlsm_sz\tblob_sz\tc_wgb\tw_amp\tc_mbps\tc_wsecs\tc_csecs\tb_rgb\tb_wgb\tusec_op\tp50\tp99\tp99.9\tp99.99\tpmax\tuptime\tstall%\tNstall\tu_cpu\ts_cpu\trss\ttest\tdate\tversion\tjob_id"
```

Comment:
The new column headings are even less descriptive than the old ones.  Can we make the descriptions more meaningful?

Reply:
These are tab delimited and prior to my change that header is hard to read because it isn't aligned with the data that follows. The fix is to make sure that the header fields are each less than 7 characters. The cost of this fix is that the reader might have to read the legend that is included.

Reply:
Is (or can we) it documented somewhere in more detail what the column headings mean, either in the script or on the Wiki or somewhere.  It would be good to have more information about what each column signifies, especially as some of the titles become more obscure.